pkg/ovs/configjob.go

```diff
@@ -0,0 +1,120 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ovs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openstack-k8s-operators/lib-common/modules/common/env"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/helper"
+	"github.com/openstack-k8s-operators/ovs-operator/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	ovnclient "github.com/openstack-k8s-operators/ovn-operator/api/v1beta1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ConfigJob - prepare job to configure ovn-controller
+func ConfigJob(
+	ctx context.Context,
+	h *helper.Helper,
+	k8sClient client.Client,
+	instance *v1beta1.OVS,
+	labels map[string]string,
+) ([]*batchv1.Job, error) {
+
+	var jobs []*batchv1.Job
+	runAsUser := int64(0)
+	privileged := true
```

Comment:
I wonder why it must be privileged. The job only sets some values in ovsdb, which only requires access to the db socket.

Reply:
It seems (or it's just my lack of OCP knowledge) that it needs to be like that to get access to the db file when e.g. doing "ovs-vsctl show" command.

Reply:
This can be revisited separately later if needed, I actually have it in my plate - to look into our privilege settings